protocol/status_update.go

```diff
@@ -0,0 +1,18 @@
+package protocol
+
+import "github.com/status-im/status-go/protocol/protobuf"
+
+type UserStatus struct {
+	PublicKey  string `json:"public-key,omitempty"`
```

Comment:
We are moving away from `kebab-case` (albeit very slowly :)), as `camelCase` is generally more widely accepted